docs/source/download.md

```diff
@@ -0,0 +1,66 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Download
+
+The official Apache DataFusion releases are provided as source artifacts.
```

Comment:
I think we may cause confusion if we don't also clarify how the code on crates.io related to the downloads linked from this page

```suggestion
While DataFusion is also distributed via the Rust [crates.io] package manager as a convenience, the 
official Apache DataFusion releases are provided as source artifacts. 

[crates.io]: https://crates.io/crates/datafusion
```